assets/csidriveroperators/shared-resource/05_clusterrole.yaml

```diff
@@ -0,0 +1,269 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
Does the CSI driver need access to some CR for the driver? I don't see its RBAC here. In addition, who is creating the CRD? It could be CVO (then put the CRD into `/manifests/`) or it could be your operator (then the operator needs permissions to create/update CRDs).